dotnet/src/Connectors/Connectors.OpenAI/ToolCallBehavior.cs

```diff
@@ -91,6 +93,8 @@ private ToolCallBehavior(bool autoInvoke)
     /// <summary>
     /// Options to control tool call result serialization behavior.
     /// </summary>
+    [Obsolete("This property is deprecated in favor of Kernel.SerializerOptions that will be introduced in one of the following releases.")]
```

Comment:
You'll probably need to suppress the warnings you get on the various use sites.

Reply:
Yep, suppressed.

Reply:
Ok